api-tests/features/repeat-fraud-check/p2-repeat-fraud-check.feature

```diff
@@ -1,138 +1,214 @@
 @Build
 Feature: Repeat fraud check journeys
 
-  Background:
-    Given the subject already has the following credentials
-      | CRI     | scenario                     |
-      | dcmaw   | kenneth-driving-permit-valid |
-      | address | kenneth-current              |
-    And the subject already has the following expired credentials
-      | CRI   | scenario        |
-      | fraud | kenneth-score-2 |
-    When I start a new 'medium-confidence' journey
-    Then I get a 'confirm-your-details' page response
+  Rule:
```

Comment:
Is it OK to leave this rule blank?

Reply:
That's a good question. It's fine syntax but I can see the convention might call for a descriptor but I was questioning "Main" as it's sort of everything else

Reply:
I suppose we leave Background blank